examples/angular-example/tsconfig.json

```
@@ -0,0 +1,29 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
```

Comments most likely aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- Creating another file at the same spot with the same name, but the fileending `.md`
- using a hack to define this like this (does the functionality still work?):

```suggestion
{
  "_to_learn_more_about_this_file_see": "https://angular.io/config/tsconfig",
```